src/jit/codegenxarch.cpp

```diff
@@ -2645,8 +2576,6 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
 bool
 CodeGen::genStoreRegisterReturnInLclVar(GenTreePtr treeNode)
 {
-
-
     if (varTypeIsStruct(treeNode))
```

Comment:
Isn't this the same as issue https://github.com/dotnet/coreclr/issues/2700 ( #2700  )? We discussed this earlier.